### DISCUSSION TITLE
Create Tags

### DISCUSSION THREAD

**Discussion by BroCrows** [WolfDen133/WFT-OLD#5] [Ideas]
As said in #2!
Add these tags! Any order you want

# Plugins I Use
OnlineTime - {onlinetime.total.x} 

# Other Plugins
KDR - {kills.x} {kdr.kdr.x}
BravoClan - {clan.lvl.x} & {clan.kills.x}
FactionsPro/FactionsPE/PiggyFactions - {faction}
Skyblock/RedSkyblock - {is.rank.x} & or {is.value.x}
Prisons - {prisons.rank.x}
Bedwars - {bedwars.kills.x} & {bedwars.wins.x}
Skywars - {skywars.kills.x} & {skywars.wins.x}
EconomyAPI - {economy.top.x}
BetterVoting/PocketVote - {topvoter.x} (kinda useless as for the plugin, TopVoter)

**Comment by WolfDen133** [comment]
Your feedback has been taken into account and you will probably see these in some form :)
